pkg/querier/blocks_store_queryable_test.go

```diff
@@ -2009,3 +2009,21 @@ func valuesFromSeries(name string, series ...labels.Labels) []string {
 	sort.Strings(values)
 	return values
 }
+
+func TestBlocksStoreQueryableErrMsgs(t *testing.T) {
+	tests := map[string]struct {
+		err error
+		msg string
+	}{
+		"newStoreConsistencyCheckFailedError": {
+			err: newStoreConsistencyCheckFailedError([]ulid.ULID{ulid.MustNew(1, nil)}),
+			msg: `consistency check failed because some blocks were not queried (err-mimir-store-consistency-check-failed). The blocks are: 00000000010000000000000000`,
```

Comment:
```suggestion
			msg: `The consistency check failed because some blocks were not queried (err-mimir-store-consistency-check-failed). The blocks are: 00000000010000000000000000`,
```

Reply:
Applied except for the initial capital